### ISSUE TITLE
Issue with LetsEncrypt expiration

### ISSUE THREAD

**Issue by ngzhongcai** [goruck/alexa-ip-cam#20]
I am using Echo Show 10 (3rd Gen). Sometimes back, out of the blue, Alexa couldn't connect to my RTSP feed. I wonder if this is related to the issue that you described on the home page. Am hoping I can create an issue here, just so to keep track of when issue is resolved. Thank you for the good work! =)

**Comment by goruck** [comment]
@ngzhongcai good idea. Latest update is Amazon has root caused and fixed the issue. Its in QA but has not yet been deployed to the field. In the meantime, the ```alternative chain``` workaround should work for you as described in the Readme.
